Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/callback/cxf/CONNECTSamlAssertionValidator.java

```diff
@@ -65,7 +68,7 @@
 public class CONNECTSamlAssertionValidator extends SamlAssertionValidator {
 
     /** The Constant LOG. */
-    private static final Logger LOG = Logger.getLogger(CONNECTSamlAssertionValidator.class);
+    private static final Logger LOG = LoggerFactory.getLogger(CONNECTSamlAssertionValidator.class);
```

Comment:
Change this back to to log4j.  I don't think the SLF4J change was made during 4.5.